src/client/entity-editor/creator-section/creator-section.js

```diff
@@ -284,11 +284,13 @@ function mapStateToProps(rootState, {creatorTypes}: OwnProps): StateProps {
 	}
 	const group = typeValue === groupType.id;
```

Comment:
Could you please rename the variable to `isGroup`like you did in the entities/creator.js page?
I think it's a lot clearer than just group (that might have been me who wrote that, and it's not pretty :) )

And could you apply the same naming in the `labelsForCreator` utility as well please?